.github/workflows/eslint.yml

```diff
@@ -0,0 +1,21 @@
+name: javascript
+on: push
+jobs:
+    build:
+      name: eslint
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v1
+        - uses: actions/setup-node@v1
+          with:
+            node-version: '10.x'
+
+        - name: yarn install
+          run: yarn install
+
+        - name: eslint
+          uses: getsentry/action-eslint@v1
```

Comment:
Why do we have a custom action?

Reply:
I think we want tighter control over the actions, didn't want it from changing under us.

Reply:
Other orgs do this as well.